lib/Ix/DBIC/ResultSet.pm

```diff
@@ -906,6 +1005,16 @@ sub get_collection_lock ($self, $ctx) {
   };
 }
 
+=method ix_set($ctx, $arg)
+
+The method implemeents 'Foo/set'. It does some initial error checking and
+validation and calls any pre-set hooks. It then runs, in turn, C<ix_destroy>,
+C<ix_update>, and C<ix_create> to handle the actual database operations for
+the/set call. It calls any postprocessing hooks, then returns a list of
+L<Ix::Result::FoosSet> result objects.
```

Comment:
Maybe some discussion about why we settled on this order of operations (destroy before update before create)

Reply:
Sure, have copy/pasted your inline comment from below here and reworded slightly.